examples/port-2-alibaba2-connection/main.tf

```diff
@@ -0,0 +1,65 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+provider "alicloud" {
+  access_key = var.access_key
+  secret_key = var.secret_key
+  region = var.region
+}
+
+resource "random_string" "random" {
+  length           = 12
+  special          = false
+}
+
+module "create_port_2_alibaba_connection" {
+  source = "../../modules/port-connection"
+
+  connection_name       = "${var.connection_name}${random_string.random.result}"
+  connection_type       = var.connection_type
+  notifications_type    = var.notifications_type
+  notifications_emails  = var.notifications_emails
+  bandwidth             = var.bandwidth
+  purchase_order_number = var.purchase_order_number
+  project_id            = var.project_id
+
+  # A-side
+  aside_port_name = var.aside_port_name
+  aside_vlan_tag  = var.aside_vlan_tag
+
+  # Z-side
+  zside_ap_type               = var.zside_ap_type
+  zside_ap_authentication_key = var.zside_ap_authentication_key
+  zside_ap_profile_type       = var.zside_ap_profile_type
+  zside_location              = var.zside_location
+  zside_seller_region         = var.zside_seller_region
+  zside_sp_name               = var.zside_sp_name
+}
+
+data "alicloud_express_connect_physical_connections" "nameRegex" {
```

Comment:
The main.tf should have these commented out to match the documentation's first step. We're not following our own instructions.

Reply:
Addressed with commit: [e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb](https://github.com/equinix/terraform-equinix-fabric/pull/155/commits/e2c6a1aff16a60d8cd9bdf5a6c1d4a2e5eb755cb)